kuwala/core/backend/app/routers/data_catalog.py

```diff
@@ -0,0 +1,17 @@
+from database.crud.data_catalog import get_data_catalog_items
+from database.database import get_db
+from database.schemas.data_catalog import DataCatalogItem
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+router = APIRouter(
+    prefix="/data_catalog",
+    tags=["data_catalog"],
+)
```

Comment:
It is best to use dash (/data-catalog) than underscore see [Source](https://stackoverflow.com/questions/70551890/route-url-which-is-recommended-to-use-underscore-or-dash)

Reply:
Good to know! Will change that.